packages/docs/package.json

```diff
@@ -4,7 +4,7 @@
   "private": true,
   "scripts": {
     "build": "astro build",
-    "dev:watch": "astro dev"
+    "dev:watch": "astro dev --base player-sdk/"
```

Comment:
I run the dev build with the same base to have consistent URLs between dev and production.